threedi_scenario_downloader/downloader.py

```diff
@@ -315,13 +344,16 @@ def download_raster(
     bbox=None,
     time=None,
     pathname=None,
-    is_threedi_scenario=True,  # For lizard rasters that are not a Threedi result.
+    is_threedi_scenario=True,  # For rasters that are not a Threedi result.
```

Comment:
Ook het oorspronkelijke commentaar is al wat raar en het nieuwe ook: zet je `is_threedi_scenario` nou op True als het *geen* threedi resultaat is?